docs/blog/posts/multilingual-support.md

```diff
@@ -0,0 +1,28 @@
+---
+authors:
+  - Carlos
+category:
+  - Database
+  - API
+date: 2024-04-19
+---
+# Namaste Multilingual support
+
+We are excited to announce a significant enhancement to our web application: _multilingual content management_. This update sets stone to one of our key features which is to enable multlilingual model support. This update will make managing contents in different languages easier.
+
+<!-- more -->
+
+## What's new?
+
+Multilingual content support changes the way contents are managed. Now contents can have many languages version, up to the number of languages supported. Allowing contents to have different language versions will help enhancing the accessibility of contents by breaking the language barrier. Users will be able to ask questions in their local language and receive answers in the same language, improving user experience. Here is the new look of our projet's Admin App.
+
+| Old UI                                                                | Multilingual UI                                                                |
+| --------------------------------------------------------------------- | ------------------------------------------------------------------ |
+| Main page                                                             |
+| ![old UI main screenshot](../images/mui_switch/mui_main.png) | ![Multilingual main screenshot](../images/multilingual/mult_main.png) |
+| Edit page                                                             |
+| ![old UI edit screenshot](../images/mui_switch/mui_edit.png) | ![Multilingual edit screenshot](../images/multilingual/mult_edit.png) |
```

Comment:
Could you upload higher resolution screenshots here? It shows up as small on the blogpost because it's too low res I think

Reply:
Ok Let me check how I can do that.

Reply:
If you fullscreen the webpage and take a screenshot that should be good enough